src/shared/util.js

```diff
@@ -51,6 +51,13 @@ const AnnotationMode = {
   ENABLE_STORAGE: 3,
 };
 
+const AnnotationEditorPrefix = "pdfjs_internal_editor_";
+
+const AnnotationEditorType = {
+  NONE: 0,
+  FREE_TEXT: 1,
+};
+
```

Comment:
Now I cannot help looking at the pre-existing `AnnotationType` and wondering if we couldn't just use that one here as well (to avoid a small amount of duplication).

The only thing that one is missing is a `NONE`-type, but we could possibly add that there?
However, I do recall that `AnnotationType` was formatted to follow this list: https://web.archive.org/web/20220309040754if_/https://www.adobe.com/content/dam/acom/en/devnet/pdf/pdfs/PDF32000_2008.pdf#G11.2096274 so maybe that's not great?

@timvandermeij Since I believe that you originally added the list of `AnnotationType` types, would you object to adding a non-standard `NONE: 0`-entry to it to avoid having to add a new and very similar list here?

Reply:
Maybe for now, this enum should reflect what is implemented, wdyt ?

Reply:
> Maybe for now, this enum should reflect what is implemented, wdyt ?

Sure, this would obviously be clearer w.r.t. what's actually supported.